src/Analyzers/CSharp/CodeFixes/InvokeDelegateWithConditionalAccess/InvokeDelegateWithConditionalAccessCodeFixProvider.cs

```diff
@@ -38,7 +38,7 @@ public InvokeDelegateWithConditionalAccessCodeFixProvider()
         // Filter out the diagnostics we created for the faded out code.  We don't want
         // to try to fix those as well as the normal diagnostics we created.
         protected override bool IncludeDiagnosticDuringFixAll(Diagnostic diagnostic)
```

Comment:
Based on the comment above, this method intended to filter out special `Unnecessary` diagnostics created for fading. Now that we don't have these diagnostics, can we remove this method completely?

Reply:
I am also wondering if there is a test gap here - your current change would filter out the primary diagnostic with additional faded location, so FixAll will basically just fix the primary diagnostic?

Reply:
This one is strange. We do still have the extra diagnostics, and several tests covering their usage.

Reply:
Prior uses of `UnnecessaryWithoutSuggestionDescriptor` do not need additional diagnostics. However, this diagnostic used `UnnecessaryWithSuggestionDescriptor` to ensure that the code fix is presented on multiple lines. It's possible we could change this in the future, but for the initial pull request I preserved this behavior.